src/App.jsx

```diff
@@ -14,33 +15,83 @@ export const goodsFromServer = [
   'Garlic',
 ];
 
-export const App = () => (
-  <div className="section content">
-    <div className="buttons">
-      <button type="button" className="button is-info is-light">
-        Sort alphabetically
-      </button>
+export const App = () => {
+  const [list, setList] = useState(goodsFromServer);
+  const [activeSort, setActiveSort] = useState(null);
 
-      <button type="button" className="button is-success is-light">
-        Sort by length
-      </button>
+  const sortList = type => {
+    setActiveSort(type);
+    const listCopy = [...list];
 
-      <button type="button" className="button is-warning is-light">
-        Reverse
-      </button>
+    if (type === 'reset') {
+      setList(goodsFromServer);
+      setActiveSort(null);
 
-      <button type="button" className="button is-danger is-light">
-        Reset
-      </button>
-    </div>
+      return;
+    }
+
+    switch (type) {
+      case 'reverse':
+        listCopy.reverse();
+        break;
+      case 'alphabetically':
+        // listCopy.sort((a, b) => a.localeCompare(b));
+        listCopy.sort((a, b) => b.localeCompare(a));
+        break;
+      case 'length':
+        listCopy.sort((a, b) => a.length - b.length);
+        break;
+      default:
+        break;
```

Comment:
It is best to make a return here

```suggestion
      default:
        return;
```